wandb/sdk/launch/launch.py

```diff
@@ -71,7 +71,9 @@ def _run(
     runner_config: Dict[str, Any] = {}
     runner_config[PROJECT_SYNCHRONOUS] = synchronous
     runner_config[PROJECT_DOCKER_ARGS] = (
-        launch_config["docker"] if launch_config else {}
+        launch_config.get(launch_config.get("docker", {}).get("args", None), {})
```

Comment:
this line is super confusing can we break this up lol

Reply:
good call